docker-compose.yml

```diff
@@ -7,12 +7,12 @@ services:
       context: .
     image: hmcts/ccd-definition-designer-api
     environment:
-      - DEFINITION_STORE_DB_HOST=ccd-definition-store-database
-      - DEFINITION_STORE_DB_PORT=5432
-      - DEFINITION_STORE_DB_USERNAME
-      - DEFINITION_STORE_DB_PASSWORD
-      - DEFINITION_STORE_IDAM_KEY
-      - DEFINITION_STORE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_gw
+      - DEFINITION_DESIGNER_DB_HOST=ccd-definition-store-designer
+      - DEFINITION_DESIGNER_DB_PORT=5432
+      - DEFINITION_DESIGNER_DB_USERNAME
+      - DEFINITION_DESIGNER_DB_PASSWORD
+      - DEFINITION_DESIGNER_IDAM_KEY
+      - DEFINITION_DESIGNER_S2S_AUTHORISED_SERVICES=ccd_data,ccd_gw
```

Comment:
@adr1ancho - shouldn't you remove `ccd_data` and add `ccd_admin`? At least, that's what's in the revised documentation...